src/features/donations/components/ShareOptions.jsx

```diff
@@ -145,4 +151,11 @@ const ShareOptions = (props) => {
   ) : null;
 };
 
+ShareOptions.propTypes = {
+  treeCount: PropTypes.string.isRequired,
+  sendRef: PropTypes.node.isRequired,
+  handleTextCopiedSnackbarOpen: PropTypes.func,
+  contactDetails: PropTypes.node
```

Comment:
Prop types of node means we can pass anything that can be rendered, such as numbers, string, DOM elements, arrays, or fragments. why not use array since we are expecting an array.

Reply:
Ok, I wasn't sure what kind of object contactDetails might be.